cpu/cpu.go

```diff
@@ -112,7 +113,7 @@ func calculateBusy(t1, t2 TimesStat) float64 {
 	if t2All <= t1All {
 		return 1
```

Comment:
Noticed during fixing this, shouldn't it be `return 100` @shirou @vrecan? This comes from https://github.com/shirou/gopsutil/commit/c3899894530a9b38dcad8ceca651ec44895f8441

Reply:
exactly! I will fix in another PR.